api.swagger.yaml

```diff
@@ -5421,6 +5421,82 @@ paths:
           description: The user does not have access to the requested content.
         '404':
           description: No content with that UUID was found.
+  '/follows/{content_id}':
+    parameters:
+      - name: content_id
+        description: The Invotra UUID of content/comment.
```

Comment:
For following you can either follow content or a user. Could we update the descriptions to reflect this?